pkgs/tools/misc/graph-easy/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPerlPackage, fetchurl }:
+
+buildPerlPackage rec {
+  name = "Graph-Easy-${version}";
+  version = "0.76";
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/S/SH/SHLOMIF/${name}.tar.gz";
+    sha256 = "d4a2c10aebef663b598ea37f3aa3e3b752acf1fbbb961232c3dbe1155008d1fa";
+  };
+  
+  meta = with stdenv.lib; {
+    homepage = http://search.cpan.org/~tels/Graph-Easy/bin/graph-easy;
+    description = "render/convert graphs in/from various formats";
```

Comment:
Please capitalize `description`

Reply:
Will do.
Other question, should this package move to pkgs/tools/graphics ?

Reply:
If you think it's appropriate to move it, sure; I think you're better suited to judge that than I am in any case.

Reply:
Moved it over to tools/graphics. Makes more sense.